cwr/convert.go

```diff
@@ -228,6 +234,39 @@ func newRecord(line string) (*Record, error) {
 		record.TransactionSequenceN = substring(line, 3, 12)
 		record.RecordSequenceN = substring(line, 12, 19)
 		record.PublisherSequenceNumber = substring(line, 19, 21)
+		record.InterestedPartyNumber = substring(line, 21, 30)
+		record.PublisherName = substring(line, 30, 75)
+		record.PublisherUnknownIndicator = substring(line, 75, 76)
+		record.PublisherType = substring(line, 76, 78)
+		record.TaxIDNumber = substring(line, 78, 87)
+		record.PublisherIPINameNumber = substring(line, 87, 98)
+		record.SubmitterAgreementNumber = substring(line, 98, 112)
+		record.PRAffiliationSocietyNumber = substring(line, 112, 115)
+		record.PROwnershipShare = substring(line, 115, 120)
+		record.MRSociety = substring(line, 120, 123)
+		record.MROwnershipShare = substring(line, 123, 128)
+		record.SRSociety = substring(line, 128, 131)
+		record.SROwnershipShare = substring(line, 131, 136)
+		record.SpecialAgreementsIndicator = substring(line, 136, 137)
+		record.FirstRecordingRefusalInd = substring(line, 137, 138)
+		record.PublisherIPINameNumber = substring(line, 139, 152)
+		record.InterStandardAgreementCode = substring(line, 152, 166)
+		record.SocietyAssignedAgreementNumber = substring(line, 166, 180)
+		record.AgreementType = substring(line, 180, 182)
+		record.USALicenseInd = substring(line, 182, 183)
+	case "SWR", "OWR":
+		record.RecordType = substring(line, 0, 3)
+		record.TransactionSequenceN = substring(line, 3, 12)
+		record.RecordSequenceN = substring(line, 12, 19)
+		record.InterestedPartyNumber = substring(line, 19, 28)
+		record.WriterLastName = substring(line, 28, 73)
+		record.WriterFirstName = substring(line, 73, 103)
```

Comment:
Should these strings be trimmed of trailing whitespace?

Reply:
I guess it should be trimmed like all others fields

Reply:
update at 7d2774d